composer/algorithms/fused_layernorm/README.md

```diff
@@ -0,0 +1,70 @@
+# 🍰 Fused LayerNorm
```

Comment:
Thoughts about using the fondue emoij? lol
🫕

Reply:
Why fondue? Because it is fused? 

I'm not very opinionated on the emoji... I chose cake because of the "layers", and "LayerNorm"

I also defer to the emoji master himself, @hanlint

Reply:
Founde is very fused haha; I'm missing the pun with cake.

Reply:
Cake is good :)